stats/metrics.go

```diff
@@ -4,197 +4,170 @@
+	if err != nil {
+		log.Fatal().Err(err).Msgf("Failed to create Prometheus exporter")
 	}
+	provider := metric.NewMeterProvider(metric.WithReader(exporter))
+	meter := provider.Meter("l3af-project/l3afd")
 
-	NFRunning = nfRunningVec.MustCurryWith(prometheus.Labels{"host": hostname})
+	baseAttribs := []attribute.KeyValue { attribute.Key("host").String(hostname) }
 
-	nfStartTimeVec := prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Namespace: daemonName,
-			Name:      "NFStartTime",
-			Help:      "This value indicates start time of the network function since unix epoch in seconds",
-		},
-		[]string{"host", "ebpf_program", "direction", "interface_name"},
-	)
+	metricName := daemonName + "_NFStartCount"
+	startCount, err := meter.Int64Counter(metricName, api.WithDescription("The count of network functions started"))
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to create NFStartCount")
+	}
+	NFStartCount = NewCounterValue(ctx, &startCount, metricName, baseAttribs)
 
-	if err := prometheus.Register(nfStartTimeVec); err != nil {
-		log.Warn().Err(err).Msg("Failed to register NFStartTime metrics")
+	metricName = daemonName + "_NFStopCount"
+	stopCount, err := meter.Int64Counter(metricName, api.WithDescription("The count of network functions stopped"))
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to create NFStartCount")
 	}
+	NFStopCount = NewCounterValue(ctx, &stopCount, metricName, baseAttribs)
 
-	NFStartTime = nfStartTimeVec.MustCurryWith(prometheus.Labels{"host": hostname})
+	metricName = daemonName + "_NFUpdateCount"
+	updateCount, err := meter.Int64Counter(metricName, api.WithDescription("The count of network functions updated"))
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to create NFUpdateCount")
+	}
+	NFUpdateCount = NewCounterValue(ctx, &updateCount, metricName, baseAttribs)
 
-	nfMonitorMapVec := prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Namespace: daemonName,
-			Name:      "NFMonitorMap",
-			Help:      "This value indicates network function monitor counters",
-		},
-		[]string{"host", "ebpf_program", "map_name", "interface_name"},
-	)
+	gaugeValues := []*GaugeValue{}
 
-	if err := prometheus.Register(nfMonitorMapVec); err != nil {
-		log.Warn().Err(err).Msg("Failed to register NFMonitorMap metrics")
+	metricName = daemonName + "_NFRunning"
+	runningGugage, err := meter.Float64ObservableGauge(metricName, api.WithDescription("This value indicates network functions is running or not"))
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to create NFRunning")
 	}
+	NFRunning = NewGaugeValue(&runningGugage, metricName, baseAttribs)
+	gaugeValues = append(gaugeValues, NFRunning)
 
-	NFMonitorMap = nfMonitorMapVec.MustCurryWith(prometheus.Labels{"host": hostname})
+	metricName = daemonName + "_NFStartTime"
+	startTimeGuage, err := meter.Float64ObservableGauge(metricName, api.WithDescription("This value indicates start time of the network function since unix epoch in seconds"))
```

Comment:
```suggestion
	startTimeGauge, err := meter.Float64ObservableGauge(metricName, api.WithDescription("This value indicates start time of the network function since unix epoch in seconds"))
```